docs/specs/bug_and_feedback_submission.md

```diff
@@ -0,0 +1,56 @@
+# Tech Spec: Bug and Feedback Submission
+Let users submit bugs or feedback directly from the Suite app.
+
+**Feature Spec:** [Notion](https://www.notion.so/satoshilabs/Feedback-Bug-report-d7cfccbacf084ce3b2bb97ce8c24e8e2#ddfe5670679f4e039a598c3a35d9c3b2), **Design:** [Zeplin](https://zpl.io/aBzxkDq), **Issue:** [GitHub](https://github.com/trezor/trezor-suite/issues/3714)
+
+_note: Though we're dealing with two distinct streams here (bugs and feedback) the technical infrastructure will be mostly identical. Hence, in this spec, we'll call bugs and feedback collectively as messages._
+
+## Scope
+At on side we have the Suite application that can send HTTP requests. On the other side we use [Keboola](https://www.keboola.com/) as the central hub for all data collected around Trezor. This spec describes how we connect these two sides.
+
+## Objectives
+1. Messages contain mostly structured textual data and sometimes can contain an image.
+2. Messages need to be consumed by Keboola.
+3. The submitted messages must be kept private as they might contain sensitive user information. Only after going thru a manual triage they might end up in a publicly accessible place like, for example, GitHub issues.
+4. The messages should be enriched with a timestamp and client's IP on the server side for client's clock might be off and determining public IP on client side requires an external service anyway.
```

Comment:
Good question. The IP indeed isn't needed for the sole purpose of collecting bugs or feedback.

When I'm thinking about it now: IP *might* become useful in case someone will decide to flood our 'API' with fake bug reports as common identifier of those fake reports. Still, this is quite hypothetical scenario and I personally would prefer to not collect IPs as well.

Reply:
Yes, let's not do it until we have a clear and justifiable reason to do so.

Reply:
Removed in 0bd3efc2761d8ff96dca6c0ef57764c4ac3059b8